src/db/connection.py

```diff
@@ -19,9 +19,13 @@ def execute_query(query, vars=None):
     """returns a cursor from the executed query"""
```

Comment:
execute query is a convenience function when only one or a few queries need to be executed. The base model uses the cursor directly because it needs to do a series of operations. Same with the per-row csv uploading within the controller